Gems/Terrain/Registry/system_materials.setreg

```diff
@@ -0,0 +1,18 @@
+{
+    "O3DE": {
+        "Atom": {
+            "Tools": {
+                "EditablePathSettings": {
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/DefaultPbrTerrain.material": false,
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/PbrTerrain.materialtype": false,
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/TerrainDetailMaterial.materialtype": false
+                },
+                "PreviewablePathSettings": {
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/DefaultPbrTerrain.material": false,
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/PbrTerrain.materialtype": false,
+                    "@gemroot:Terrain@/Assets/Materials/Terrain/TerrainDetailMaterial.materialtype": false
+                }
```

Comment:
TerrainDetailMaterial ought to remain editable and previewable. It contains additional fields for height blending beyond StandardPBR that can't be previewed, but the base material should be previewable.